docs/assets/scripts/example-knn-image.js

```diff
@@ -4,93 +4,67 @@ KNN Image Classifier Demo
 ===
 */
 
-let knn;
-let video;
+// Set up the video stream
+const video = document.getElementById('video');
+navigator.getUserMedia({ video: true }, handleVideo, videoError);
```

Comment:
Let's keep p5 for this example!

Reply:
There should probably be documentation somewhere (maybe an inline use case in the p5 api docs) of using this with vanilla JS too.